### DISCUSSION TITLE
Custom Empire and System Properties

### DISCUSSION THREAD

**Discussion by Clashsoft** [sekassel/stp-24-server-issues#14] [General]
Please comment your use of the `Empire` and `System` custom `_public` and `_private` properties.
Please use the following format:

| Kind of Ressource | Public/Private | Property | Type | Purpose |
|--------|--------|--------|--------|--------|
| Empire | private | `UmbraEvents` | `List<Object>` | Events that are active or currently starting |

For copy & paste:

```md
| Kind of Ressource | Public/Private | Property | Type | Purpose |
|--------|--------|--------|--------|--------|
| Empire | public | `thingamabobs` | `number` | the number of thingamabobs produced |
| System | public | `previousOwner` | `string` | previous owner |
| Empire | private | `customResources` | `object` | map of custom resources |
```

**Comment by arthur-flaum** [comment]
| Kind of Ressource | Public/Private | Property | Type | Purpose |
|--------|--------|--------|--------|--------|
| Empire | private | `UmbraEvents` | `List<Object>` | Events that are active or currently starting |

**Comment by APineiro-Uni** [comment]
| Kind of Ressource | Public/Private | Property | Type | Purpose |
|--------|--------|--------|--------|--------|
| Empire | private | `crisis` | `Object` | Saves the settings for the crisis |
